osu.Game/Online/API/Requests/GetBeatmapsRequest.cs

```diff
@@ -7,7 +7,7 @@ namespace osu.Game.Online.API.Requests
 {
     public class GetBeatmapsRequest : APIRequest<GetBeatmapsResponse>
     {
-        private readonly int[] beatmapIds;
+        public readonly int[] BeatmapIds;
```

Comment:
Just going to note that a `public readonly int[]` is potentially mutable (you can change the elements). Nobody should ever be doing this, but maybe worth locking down by scoping to `IReadOnlyList` or similar...?